code/modules/projectiles/ammunition/ammo_casings.dm

```diff
@@ -270,11 +281,6 @@
 	projectile_type = /obj/item/projectile/bullet/dart/syringe/tranquilizer
 	materials = list(MAT_METAL=250)
 
-/obj/item/ammo_casing/a556
-	desc = "A 5.56mm bullet casing."
-	caliber = "a556"
-	projectile_type = /obj/item/projectile/bullet/heavybullet
```

Comment:
Nothing else uses `/obj/item/projectile/bullet/heavybullet` so you may as well also remove this object if you're removing this.

Reply:
I would, but there are things that still use heavybullet2, and someone might want to at some point use it, but I do think it would be kinda funny to start at heavybullet2, and confuses everyone who tries to code ammo later on.